theme/material/vaadin-grid-styles.html

```diff
@@ -121,6 +121,88 @@
       :host([navigating]) [part~="cell"]:focus {
         box-shadow: inset 0 0 0 2px var(--material-primary-color);
       }
+
+      /* Drag and Drop styles */
+      :host([dragover])::after {
+        content: "";
+        position: absolute;
+        z-index: 100;
+        top: 0;
+        right: 0;
+        bottom: 0;
+        left: 0;
+        pointer-events: none;
+        box-shadow: inset 0 0 0 2px var(--material-primary-color);
+      }
+
+      [part~="row"][dragover] {
+        z-index: 100 !important;
+      }
+
+      [part~="row"][dragover] [part~="cell"] {
+        overflow: visible;
+      }
+
+      [part~="row"][dragover] [part~="cell"]::after {
+        content: "";
+        position: absolute;
+        top: 0;
+        right: 0;
+        bottom: 0;
+        left: 0;
+        height: 3px;
```

Comment:
**Suggestion:** something like `--lumo-size-xs / 8` ?

Reply:
Shouldn't use lumo variables in material theme